### PR TITLE
Feat: redesign animes slider

### PR DESCRIPTION
**Pull request by moonlitgrace**

### CODE REVIEW

File: src/lib/components/dominant_color/image-worker.ts [hunk 1]
```diff
@@ -10,10 +10,10 @@ onmessage = async function (event) {
 		dominant_color =
 			chroma(color_palette[0]).get("lab.l") < 40
 				? chroma(color_palette[0]).brighten().hex()
-				: chroma(color_palette[0]).hex(),
+				: chroma(color_palette[0]).darken(2).hex(),
 		dominant_foreground_color =
 			chroma(color_palette[0]).get("lab.l") < 40
-				? chroma(color_palette[0]).brighten(2).hex()
+				? chroma(color_palette[0]).brighten(3).hex()
 				: chroma(color_palette[0]).hex();
```
Comment (baseplate-admin) [inline src/lib/components/dominant_color/image-worker.ts:17]: This is being touched too much

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/lib/components/dominant_color/image-worker.ts (hunk 1 above)
Comment (moonlitgrace) [reply src/lib/components/dominant_color/image-worker.ts:17]: Yea, well we need to return variety of shades of dominant color.
return more shades?

Comment (baseplate-admin) [reply src/lib/components/dominant_color/image-worker.ts:17]: I do understand it, but i feel like you are touching it too much. Maybe use chroma  for each item?

Comment (moonlitgrace) [reply src/lib/components/dominant_color/image-worker.ts:17]: > Maybe use chroma for each item?

You want to use chrome outside?

Comment (baseplate-admin) [reply src/lib/components/dominant_color/image-worker.ts:17]: > You want to use chrome outside?

Sure

**Pull request merged**